src/DeviceInterface/InterfacePrototype.js

```diff
@@ -464,7 +466,8 @@ class InterfacePrototype {
 
         /** @type {TopContextData} */
         const topContextData = {
-            inputType
+            inputType,
+            credentialsImport: form.shouldShowCredentialsImportFlow
```

Comment:
You shouldn't need to reach for the `form` here. That method should be available from inside this interface at `this.device.CredentialsImport`, isn't it?

Reply:
The only thing is if we check availableInputTypes in top frame, I think it isn't valid. But I will re-validate that idea. This part was a while ago